vmlDrawing.go

```diff
@@ -193,15 +193,19 @@ type decodeVmlDrawing struct {
 // decodeShapeType defines the structure used to parse the shapetype element in
 // the file xl/drawings/vmlDrawing%d.vml.
 type decodeShapeType struct {
-	ID        string `xml:"id,attr"`
-	CoordSize string `xml:"coordsize,attr"`
-	Spt       int    `xml:"spt,attr"`
-	Path      string `xml:"path,attr"`
+	ID             string `xml:"id,attr"`
+	CoordSize      string `xml:"coordsize,attr"`
+	Spt            int    `xml:"spt,attr"`
+	PreferRelative string `xml:"preferrelative,attr,omitempty"`
+	Path           string `xml:"path,attr"`
+	Filled         string `xml:"filled,attr,omitempty"`
+	Stroked        string `xml:"stroked,attr,omitempty"`
 }
 
 // decodeShape defines the structure used to parse the particular shape element.
 type decodeShape struct {
 	ID          string `xml:"id,attr"`
+	SpID        string `xml:"spid,attr,omitempty"`
```

Comment:
`o:spid`, no?

Reply:
This library using two structs to resolve limitation XML name space support of Go encoding/xml, no need namespace in the `decode` prefix structures.